### DISCUSSION TITLE
SASS media query not working on NextJs website. Plz Help!

### DISCUSSION THREAD

**Discussion by devanup** [vercel/next.js#22360] [Help]
I am a new Next JS learner. I am creating a website as a practice, but I am having problem making the website responsive. I am using SCSS but the media query isn't working.
Example: I want to customize the header for mobile view so when I do

@media (max-width: 300px) {
	.header {
		// styling ...
	}
} 

There is no change on the header. I am not sure why ? I also have tried doing 

.header {
	@media (max-width: 300px) {
			// styling ...                            
         }
}

I still see no change on the header. I am stuck. Can anyone please help me?

![Screen Shot 2021-02-19 at 8 03 47 PM](https://user-images.githubusercontent.com/66684609/108577116-e5041e00-72ed-11eb-9bcf-f6a76235a0a5.png)

**Comment by Hermestech** [comment]
hey dude  did you resolve this?
